docs/book/src/development/common.md

```diff
@@ -31,7 +31,7 @@
     # Referring to a prerequisite capi-compatible image you've loaded into Apache CloudStack
     export CLOUDSTACK_TEMPLATE_NAME=kube-v1.20.10/ubuntu-2004
 
-    # The SSH KeyPair to log into the VM (Optional)
+    # The SSH KeyPair to log into the VM (Optional: use flavor *managed-ssh*)
```

Comment:
speaking of flavors, I believe the template files currently are all based on the v1beta1 schema. Should we also bump them to the v1beta2 schema and explain how we're using failuredomains to support multi-endpoint somewhere?

Reply:
Template v1beta2 bump is in a separate [PR-158](https://github.com/kubernetes-sigs/cluster-api-provider-cloudstack/pull/158).

Agreed that failuredomains are worthy of documentation coverage.